src/Form/DatasetSubmissionType.php

```diff
@@ -273,6 +273,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'mapped' => false,
                 'required' => false,
             ))
+            ->add('coldStorageTotalUnpackedCount', Type\IntegerType::class, array(
+                'label' => 'Cold Storage Total Unpacked File Count',
+                'mapped' => false,
```

Comment:
We have set all the other ones to False too. It is set to false when the form is being processed, because these items on not usually on the submission form.